src/modules/feature-modules/accessor/pages/innovation/needs-assessment-overview/needs-assessment-overview.component.ts

```diff
@@ -28,7 +28,7 @@ export class InnovationNeedsAssessmentOverviewComponent extends CoreComponent im
   innovation: InnovationDataResolverType;
 
   assessment: getInnovationNeedsAssessmentEndpointOutDTO['assessment'] | undefined;
-  suggestedOrganisations: string[] = [];
+  suggestedOrganisations: { [key: string]: any }[] = [];
```

Comment:
This should be strongly typed. "any" should only be used in very specific circumstances.